okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -387,11 +385,11 @@ public void run() {
           }
           clientFrameHandler = new ClientFrameHandler(testFrameReader);
           executor.execute(clientFrameHandler);
+          frameWriter.becomeConnected(testFrameWriter, socket);
```

Comment:
Why is triggering sending a problem? Everything should be queued behind this currently-running Runnable.